src/datepicker/datepicker.ts

```diff
@@ -41,6 +41,7 @@ const NGB_DATEPICKER_VALUE_ACCESSOR = {
         [dayTemplate]="dayTemplate || dt"
         [showWeekdays]="showWeekdays"
         [showWeekNumbers]="showWeekNumbers"
+        [disabled]="disabled"
```

Comment:
I wasn't sure, so I chose the easy route :-)

But yes, that's another thing we have to decide. It isn't strictly necessary, but it might make things clearer. If we choose to do it, I guess it would also be a good idea to switch to the month of the selected date when it becomes disabled.